Package.swift

```diff
@@ -2,23 +2,64 @@
 
 import PackageDescription
 
+let MeiliSearch = "MeiliSearch"
+let MeiliSearchCore = "MeiliSearchCore"
+let MeiliSearchNIO = "MeiliSearchNIO"
+
+var products: [PackageDescription.Product] = [
+  .library(name: MeiliSearch, targets: [MeiliSearch]),
+  .library(name: MeiliSearchCore, targets: [MeiliSearchCore])
+]
+
+var dependencies = [PackageDescription.Package.Dependency]()
+
+var targets: [PackageDescription.Target] = [
+  .target(
+    name: MeiliSearch,
+    dependencies: [
+      .target(name: MeiliSearchCore)
+    ]
+  ),
+  .target(
+    name: MeiliSearchCore, 
+    dependencies: []
+  ),
+  .testTarget(
+    name: "MeiliSearchUnitTests",
+    dependencies: ["MeiliSearch"]
+  ),
+  .testTarget(
+    name: "MeiliSearchIntegrationTests",
+    dependencies: ["MeiliSearch"]
+  )
+]
+
+#if os(Linux) || USE_NIO
```

Comment:
comment: I like the use of `USE_NIO` here as it'll allow macOS clients (for working on Vapor projects locally) to build the NIO integration. When we add the NIO integration (presumably the next PR), let's document how to use this.